modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -2295,11 +2296,7 @@ else if (active) {
 
 		objectDefinition.setDBTableName(dbTableName);
 		objectDefinition.setEnableIndexSearch(enableIndexSearch);
-		objectDefinition.setEnableLocalization(
-			(FeatureFlagManagerUtil.isEnabled(
-				objectDefinition.getCompanyId(), "LPD-32050") &&
-			 objectDefinition.isModifiable()) ||
-			enableLocalization);
+		objectDefinition.setEnableLocalization(enableLocalization);
```

Comment:
@devpedrotavares if `enableLocalization` must always be true, what's the point of updating it?

Reply:
After we remove the feature flag, there will be no point. But without the feature flag, we can still turn the toggle on and off when the definition is a draft.